app/services/irs_attempts_api/tracker.rb

```diff
@@ -79,6 +82,7 @@ def hashed_cookie_device_uuid
     end
 
     def enabled?
+      puts "ENABLED? #{IdentityConfig.store.irs_attempt_api_enabled && @enabled_for_session}"
```

Comment:
we probably want to remove this, and I think it'll get flagged by a rubocop linter
```suggestion
```

Reply:
Sorry!  'twas accidental...